### PR TITLE
updated dev screen modal to quell android propTypes

### PR DESCRIPTION
**Pull request by mwarger**
## Please verify the following:
- [x] Everything works on iOS/Android
- [x] `npm test` **ava** tests pass with new tests, if relevant
- [x] `./docs/` has been updated with your changes, if relevant

## Describe your PR

On android, the onRequestClose prop is required for Modals to instruct how to handle the hardware back button.  I just implemented a quick check to quell this message and toggle the modal in the same fashion as the 'X' button.

I know things have changed for ignite 2, so let me know if there's anything I need to address.

### CODE REVIEW

**Comment by mwarger** [comment]
Testing and docs not relevant, afaik.  Works on iOS and Android.

File: packages/ignite-dev-screens/templates/DevscreensButton.js [hunk 1]
```diff
@@ -23,13 +23,18 @@ export default class DevscreensButton extends React.Component {
           <RoundedButton onPress={this.toggleModal}>
             Open DevScreens
           </RoundedButton>
-          <Modal visible={this.state.showModal} >
-            <PresentationScreen screenProps={{toggle: this.toggleModal}} />
+          <Modal
+            visible={this.state.showModal}
+            onRequestClose={
+              Platform.OS === 'android' ? this.toggleModal : undefined
```
Comment (kevinvangelder) [inline packages/ignite-dev-screens/templates/DevscreensButton.js]: I don't think the Platform check is necessary. Any platforms that support a back button will call this function and those that don't won't.

Comment (mwarger) [inline packages/ignite-dev-screens/templates/DevscreensButton.js]: Oh okay.  I wasn't sure about that.  I will make that change.  Thanks.

**Comment by skellock** [comment]
Thanks 🙏

**Pull request merged**